mesos-rxjava-example/mesos-rxjava-example-framework/README.md

```diff
@@ -0,0 +1,17 @@
+# Example framework
+
+A simple Mesos framework that demonstrates API client usage.
+
+## Running
+
+You'll need a Mesos master accessible at `$mesos_host:$mesos_port`. From the
+project root directory:
+
+```bash
```

Comment:
If we remove the `$` from the beginning of each line this will be able to be copy-pasted by a user.